src/System.IO.Compression/src/System/IO/Compression/DeflateManaged/DeflateManagedStream.cs

```diff
@@ -35,7 +35,7 @@ internal DeflateManagedStream(Stream stream, ZipArchiveEntry.CompressionMethodVa
         /// <summary>
         /// Sets up this DeflateManagedStream to be used for Inflation/Decompression
         /// </summary>
-        internal void InitializeInflater(Stream stream, bool leaveOpen, IFileFormatReader reader = null, ZipArchiveEntry.CompressionMethodValues method = ZipArchiveEntry.CompressionMethodValues.Deflate, long uncompressedSize = -1)
+        internal void InitializeInflater(Stream stream, bool leaveOpen, IFileFormatReader? reader = null, ZipArchiveEntry.CompressionMethodValues method = ZipArchiveEntry.CompressionMethodValues.Deflate, long uncompressedSize = -1)
```

Comment:
I can't see any reason at this point for this InitializeInflater method to be on its own; it's only called from one ctor.  We should just inline its contents back into the ctor above.  That will then let you avoid using the `= null` on several of the fields.

Reply:
ok